frontend/e2e/_gcweb-app.letters._index.spec.ts

```diff
@@ -8,14 +8,14 @@ test.describe('letters page', () => {
   });
 
   test('it should sort list in ascending order by date', async ({ page }) => {
-    await page.goto('/letters');
-    await page.locator('select').selectOption('asc');
+    await page.goto('/letters?sort=desc');
+    await page.locator('select#sort-order').selectOption('asc');
```

Comment:
Is it possible to test one scenario? Like get the value of the first non selected option and use to it to select the option with `.selectOption` then you can assert that the URL have a match with it?

Reply:
just testing one scenario now.  Since the default sorting is newest-to-oldest, this check will do the inverse and assert the url matches after selecting the option.